contracts/hardhat.config.js

```diff
@@ -211,12 +211,25 @@ task("showStorageLayout", "Visually show the storage layout of the contract")
 
 module.exports = {
   solidity: {
-    version: "0.8.7",
-    settings: {
-      optimizer: {
-        enabled: true,
+    compilers: [
+      {
+        version: "0.8.7",
+        settings: {
+          optimizer: {
+            enabled: true,
+          },
+        },
       },
-    },
+      {
+        version: "0.8.10",
```

Comment:
just curious why we need 2 compiler versions here?

Reply:
Several contracts from the ousd-governance repo have pragma solidity 0.8.10 - `RewardsSource`, `OgvStaking`, `OgvStakingProxy`, `RewardsSourceProxy`, `MockOgvGovernance` and the forge test contracts.

Reply:
got it thanks